student/models.py

```diff
@@ -8,6 +8,13 @@ class EducationalInstitution(models.Model):
     def __str__(self):
         return self.name
 
+    @classmethod
+    def get_institutions_with_enrolled_students(cls):
+        """
+        Return all institutions that have enrolled students as a QuerySet
```

Comment:
A function docstring should tell me WHY you wrote it - if the code is well written I can usually tell WHAT it does easily enough from the function name and the code.

Same comment goes for the code below, the function names and documentation are still not good enough to let me understand your application business logic

Reply:
Understood, once this PR gets merged I'll issue another PR to address this

Reply:
Its customary to use HitHis issues of some other issue tracking system to plan and track future work

Reply:
Will do.